README.md

```diff
@@ -141,6 +174,8 @@ format = {
                 -- unsharded spaces must NOT have
                 -- field and index named 'bucket_id'
             },
+            sharding_func = 'mpcrc32' | 'strcrc32' | 'dot.notation.func' |
+                            {body = 'function(key) return <...> end'} | <...>,
```

Comment:
The last `<...>` looks redundant and maybe even confusing.

Reply:
Updated